static/app/components/events/interfaces/crashContent/stackTrace/content.tsx

```diff
@@ -126,6 +197,17 @@ class Content extends Component<Props, State> {
     }));
   };
 
+  handleToggleFrames = (event: React.MouseEvent<SVGElement>, frameIndex: number) => {
+    event.stopPropagation(); // to prevent collapsing if collapsible
```

Comment:
Minor wording edit to make it more clear

```suggestion
    event.stopPropagation(); // to prevent toggling frame context
```